### PR TITLE
Dev/issue 13530 b5 themes treeview accordion

### PR DESCRIPTION
**Pull request by sbreker**
Treeview cleanup: wrap with b5 accordion.

### CODE REVIEW

File: plugins/arDominionB5Plugin/js/fullWidthTreeViewCollapse.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        $accordionButton.text($treeViewConfig.data("closed-text"));
+      });
+
+      // Set default Open/Close state for the treeview.
+      bootstrap.Collapse.getOrCreateInstance($accordionCollapsibleSection, {
+        toggle: !treeViewCollapseEnabled,
+      });
+    }
+  });
+})(jQuery);
```
Comment (jraddaoui) [inline plugins/arDominionB5Plugin/js/fullWidthTreeViewCollapse.js]: If possible, I'd move this to the main treeview JS where you already have the elements needed.

File: plugins/arDominionB5Plugin/js/fullWidthTreeViewCollapse.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      var treeViewCollapseEnabled =
+        $treeViewConfig.data("collapse-enabled") == "yes";
+
+      $accordionButton.text($treeViewConfig.data("closed-text"));
+
+      // Set Treeview Accordion title.
+      $treeviewAccordionItem.on("show.bs.collapse", (e) => {
+        $accordionButton.text($treeViewConfig.data("opened-text"));
+      });
+      $treeviewAccordionItem.on("hide.bs.collapse", (e) => {
```
Comment (jraddaoui) [inline plugins/arDominionB5Plugin/js/fullWidthTreeViewCollapse.js]: I'd use `shown.bs.collapse` and `hidden.bs.collapse` to make sure it happened.

**Review by jraddaoui — APPROVED** [review]
Looking great @sbreker, thanks!

Added a couple of minor commits below, but feel free to merge.

File: plugins/arDominionB5Plugin/js/fullWidthTreeViewCollapse.js (hunk 1 above)
Comment (sbreker) [reply plugins/arDominionB5Plugin/js/fullWidthTreeViewCollapse.js]: Makes sense - moved.

**Pull request merged**